src/web_app_skeleton/script/helpers/function_helpers

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+# This file contains a set of functions used as helpers
+# for various tasks. Read the examples for each one for
+# more information. Feel free to put any additional helper
+# functions you may need for your app
+
+
+# Returns true if the command $1 is not found
+# example:
+#   if command_not_found "yarn"; then
+#     echo "no yarn"
+#   fi
+command_not_found() {
+  ! command -v $1 > /dev/null
+  return $?
+}
+
+# Returns true if the command $1 is not running
+# You must supply the full command to check as an argument
+# example:
+#   if command_not_running "redis-cli ping"; then
+#     print_error "Redis is not running"
+#   fi
+command_not_running() {
+  $1
+  if [ $? -ne 0 ]; then
+    true
+  else
+    false
+  fi
+}
+
+# Returns true if the OS is macOS
+# example:
+#   if is_mac; then
+#     echo "do mac stuff"
+#   fi
+is_mac() {
+  if [[ "$OSTYPE" == "darwin"* ]]; then
+    true
+  else
+    false
+  fi
+}
+
+# Returns true if the OS is linux based
+# example:
+#   if is_linux; then
+#     echo "do linux stuff"
+#   fi
+is_linux() {
+  if [[ "$OSTYPE" == "linux"* ]]; then
```

Comment:
It seems most people I asked get back `linux-gnu` on arch, debian, or other flavors. I didn't actually test this though, but I assume it works.